uniauth/views.py

```diff
@@ -266,10 +309,16 @@ def logout(request):
         )
         client = CASClient(version=2, service_url=get_service_url(request),
                 server_url=institution.cas_server_url)
+        if api_mode:
+            # return the official logout link for the CAS institution
+            return JsonResponse({'redirect-to': client.get_logout_url(redirect_url), 'logout-method': 'CAS'})
```

Comment:
The `logoutMethod` should be `'cas'` or `'uniauth'` to be consistent with the auth-method values.